manual/manual/cmds/intf-c.etex

```diff
@@ -297,7 +297,7 @@ then loads dynamically these libraries, and resolves references to the
 required primitives, before executing the bytecode.
 
 This facility is currently supported and known to work well under
-Linux, MacOS~X, and Windows.  It is supported, but not
+Linux, MacOS, and Windows.  It is supported, but not
```

Comment:
```suggestion
Linux, macOS, and Windows.  It is supported, but not
```

minor nit -- it's actually "macOS" according to Apple's online materials.

Reply:
I stand corrected.